src/Centreon/Application/Controller/FilterController.php

```diff
@@ -132,11 +145,15 @@ public function addFilter(Request $request, string $pageName): View
      * @param int $filterId
      * @return View
```

Comment:
Please check the missing @throws : FilterException and EntityNotFoundException (https://github.com/centreon/centreon/pull/8865/files#diff-a9038fd390872e755413240c9fb99b5eR173)